modular_skyrat/modules/colony_fabricator/code/machines/rtg.dm

```diff
@@ -0,0 +1,38 @@
+/obj/machinery/power/rtg/portable
+	name = "radioisotope thermoelectric generator"
+	desc = "The ultimate in 'middle of nowhere' power generation. Unlike standard RTGs, this particular \
+		design of generator has forgone the heavy radiation shielding that most RTG designs include. \
+		In better news, these tend to be pretty good with making a passable trickle of power for any \
+		application."
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/machines.dmi'
+	circuit = null
+	flags_1 = NODECONSTRUCT_1
+	power_gen = 7500
```

Comment:
This is quite a lot for the incredibly low price that is charged to fabricate these.

I'd like to hear your answer to the following question, so it's documented:
What niche does this fill, that a PACMAN machine doesn't already, and would something like this not always be preferable to a PACMAN considering its relatively high output?

Reply:
A PACMAN has the option to produce more power, and be upgraded to produce even more if you were so inclined. What these offer that a PACMAN doesn't is that despite the genuinely quite small power production (7.5 KW is not a lot) you don't have to maintenance them. So long as you put them somewhere that the radiation they spew out doesn't matter, you can have a trickle of backup power that's not reliant on fuel or the sun.

Reply:
I think that's much better with the new printing costs.